include/glow/Runtime/RuntimeTypes.h

```diff
@@ -143,6 +143,13 @@ struct DeviceConfig {
       : backendKind(kind), name(name), parameters(parameters) {}
 
   bool hasName() const { return name != ""; }
+
+  // Device memory size.
+  uint64_t deviceMemory = 0;
```

Comment:
nit: can you group that to other members above
```
/// An enum indicating what kind of backend this config is for. It is used in
  /// checking the type of config before casting to a derived class.
  const BackendKind backendKind;
  /// A human readable name to identify the device.
  std::string name;
...
```

Also, /// not // for goxygen comment.

Reply:
> nit: can you group that to other members above
> 
> ```
> /// An enum indicating what kind of backend this config is for. It is used in
>   /// checking the type of config before casting to a derived class.
>   const BackendKind backendKind;
>   /// A human readable name to identify the device.
>   std::string name;
> ...
> ```
> 
> Also, /// not // for goxygen comment.

OK. I will do that